ASM/c/uninvertYaxis.c

```diff
@@ -9,6 +9,10 @@ void manage_uninvert_yaxis() {
         if (z64_MessageGetState(((uint8_t*)(&z64_game)) + 0x20D8) == 4) {
             return;
         }
+        // Also never in pause menu.
+        if (z64_game.pause_ctxt.state == 6) {
```

Comment:
You should move https://github.com/OoTRandomizer/OoT-Randomizer/blob/a6509aadc6454f56a6b03aa5e7fa3cf027f9a076/ASM/c/kaleido_item.h#L13-L34 to somewhere more common and use the enum instead of an integer.